cli/cmd/encore/gen.go

```diff
@@ -26,10 +26,11 @@ func init() {
 		envName          string
 		genServiceNames  []string
 		excludedServices []string
+		endpointTags     map[string]string
 	)
 
 	genClientCmd := &cobra.Command{
-		Use:   "client [<app-id>] [--env=<name>] [--services=foo,bar] [--excluded-services=baz,qux]",
+		Use:   "client [<app-id>] [--env=<name>] [--services=foo,bar] [--excluded-services=baz,qux] [--tags=cache=true,internal=false]",
```

Comment:
I liked the previous semantics better; seems more consistent with `--services` and `--excluded-services`. Can you revert this and just keep the `map[string]bool` as an implementation detail of `clientgentypes.TagSet`?

Reply:
Gotcha, done